### ISSUE TITLE
Firefox Extension not installable on Android

### ISSUE THREAD

**Issue by elmodor** [sissbruecker/linkding-extension#6]
The firefox extension:
https://addons.mozilla.org/de/firefox/addon/linkding-extension/
is not available on the new firefox version on android (firefox 88.1.3).
It says, that the extension is incompatible with firefox android on the store page.

**Comment by konstantingoretzki** [comment]
Installing and configuring the extension is possible if you use a Firefox version / fork (like Firefox Nightly or Fennec) that allows adding a custom addon collection. However it looks like the Omnibox API is not fully supported on Android. I've tried it and there aren't any results by using the `ld <search-term>` command in the search bar. 

Possible relevant issues:
- https://bugzilla.mozilla.org/show_bug.cgi?id=1427501
- https://bugzilla.mozilla.org/show_bug.cgi?id=1166831

Besides I also haven't been able to find the extension shortcut to add a bookmark.
Guess until the Omnibox API isn't fully integrated on Android it isn't any use to request Mozilla to whitelist / add the extension for Android users?

**Comment by elmodor** [comment]
Yeah I can't get it to work anymore either on Android. It worked before they reworked it.

**Comment by Nimdoc** [comment]
It is now possible to install on the latest version of Firefox on Android. To do so, you must view the [addon page](https://addons.mozilla.org/en-US/firefox/addon/linkding-extension/) in desktop mode and then install it. Then to add your credentials, you must navigate to the addon specifically through the settings menu. Then to add bookmarks, just open the menu, tap on addons, then Add Bookmark. 

It's a little clunky, but it works.

**Comment by jonathan-s** [comment]
An idea here could be to inject an icon into the page if we are on mobile, which would then popup an overlay when you click on it to save the bookmark.

**Comment by cameronj86** [comment]
> Yeah I can't get it to work anymore either on Android. It worked before they reworked it. 

Thanks for this. I was able to use desktop go to download the extension but it never occurred to me to launch the add-on manually to trigger bookmarking

**Comment by Anachron** [comment]
Is there a way how to search for bookmarks on mobile as well?

**Comment by cameronj86** [comment]
> Is there a way how to search for bookmarks on mobile as well?

If you don't mind configuring the search settings, then I believe Firefox-based browsers (works on mull) can use the following:
for keyword search:

Click the search icon in the upper left -> Search Settings -> Manage Alternative Search Engines -> Add Search Engine

`http://ipAddress:portNumber/bookmarks?q=%s`

For tag search:
`http://ipAddress:portNumber/bookmarks?q=%23%s`
